pkg/system/system.go

```diff
@@ -148,6 +148,21 @@ func NewSystemController(fs *filesystem.Filesystem, manager *manager.Manager, so
 	return &sc, nil
 }
 
+func (sc *Controller) Run() error {
```

Comment:
Um, why several functions like this `Run` are moved within the same source file, which makes the review a little harder?

Reply:
I'm trying to group logically related functions near to each other.